### PR TITLE
Study details page improvements

### PR DESCRIPTION
**Pull request by jsaq007**
## What do these changes do?
Improve the layout of the study details page based on user feedback

- Create content blocks for study details and meta data
- Fix a spacing issue where the markdown preview was cut off

Screenshot 2024-03-28 at 09 58 19

## Related issue/s

- resolves Study details page improvements #1300

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by odeimaiz — APPROVED** [review]
Please list the requested improvements in the PR's description.

Thanks!

File: services/static-webserver/client/source/class/osparc/ui/markdown/Markdown.js [hunk 1]
```diff
@@ -123,10 +123,10 @@ qx.Class.define("osparc.ui.markdown.Markdown", {
       }
       if (domElement && domElement.children) {
         const elemHeight = this.__getChildrenElementHeight(domElement.children);
-        if (this.getMaxHeight() && elemHeight > this.getMaxHeight()) {
-          this.setHeight(elemHeight);
+        if (this.getMaxHeight() && (elemHeight > this.getMaxHeight())) {
+          this.setHeight(elemHeight + 40);
```
Comment (odeimaiz) [inline services/static-webserver/client/source/class/osparc/ui/markdown/Markdown.js:127]: Where is this 40 coming from?

Comment (jsaq007) [reply services/static-webserver/client/source/class/osparc/ui/markdown/Markdown.js:127]: It seems to be padding or there is some strange behavior coming in from the markdown plugin as there isn't any code adding it from our side

**Review by GitHK — APPROVED** [review]
👍

**Review by matusdrobuliak66 — APPROVED** [review]

**Pull request merged**